documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1514,6 +1514,12 @@ twice - once in initial snapshot and once in streaming phase. Nonetheless, that
 data mirroring.
 For `read_uncommitted` there are no data consistency guarantees at all (some data might be lost or corrupted).
 
+|[[connector-property-source-timestamp-mode]]<<connector-property-source-timestamp-mode>>, `source.timestamp.mode`
+|_commit_
+|String representing the criteria of the attached timestamp within the source record (ts_ms).
+`commit` will set the source timestamp to the instant where the record was committed in the database
+`processing` will set the source timestamp to the instant where the record was processed by Debezium.
```

Comment:
Could you add a sentence describing when/why one would use `processing`?

Reply:
Sure!